README.md

```diff
@@ -81,4 +98,5 @@ git push -u origin GH-issuenumber-title-of-issue
 
 ### Pull Request
 
- When finished create a pull request from your branch to the main Simple-Static-Website repository.
+- When finished create a pull request from your branch to the main Simple-Static-Website repository.
+- When making a pull request use `Closes #(issue_number)` in the main heading of your PR, so that GitHub automatically associates your PR with that issue.
```

Comment:
This looks good, you can also mention that one can use the Web UI to link issues.

Reply:
Did that in the new commit